opencog/persist/file/fast_load.h

```diff
@@ -25,9 +25,11 @@
 
 #ifndef FAST_LOAD_H
 #define FAST_LOAD_H
+#include <opencog/atomspace/AtomSpace.h>
+#include <string>
 
 namespace opencog {
-    void load_file(std::string fname, AtomSpace& as);
+    void load_file(std::string fname, opencog::AtomSpace& as);
```

Comment:
You don't actually need to say `opencog::` here. It's already the case, since one line above, you already declare the namespace.